x-pack/plugins/security_solution/server/plugin.ts

```diff
@@ -70,6 +71,7 @@ export interface SetupPlugins {
   spaces?: SpacesSetup;
   taskManager?: TaskManagerSetupContract;
   usageCollection?: UsageCollectionSetup;
+  errorService?: Promise<ISavedObjectsRepository>;
```

Comment:
This shouldn't be part of `SetupPlugins`, `SetupPlugins` is meant to be other plugin's contracts our plugin receives during setup.

Reply:
Meaning: I would only type this on `signalRulesAlertType`